### PR TITLE
👨🏻‍🚀 Cosmos networks

### PR DESCRIPTION
**Pull request by nrjshka**

### CODE REVIEW

File: src/utils/common.ts [hunk 1]
```diff
@@ -53,6 +53,11 @@ export const isValidAddress = async (chainId: number, address: string) => {
       && /^[a-zA-Z0-9_-]*$/.test(address)
     )
   }
+
+  if (chainId === NETWORK_IDS.Cosmos || chainId === NETWORK_IDS.Osmosis || chainId === NETWORK_IDS.Sifchain) {
+    return true
+  }
+
```
Comment (nrjshka) [inline src/utils/common.ts]: Temporary solution

Comment (mgmtea) [reply src/utils/common.ts]: пока так

Comment (mgmtea) [reply src/utils/common.ts]: COSMOS ADDR REGEXP `^(cosmos1)[0-9a-z]{38}$`
OSMOSIS ADDR REGEXP `^(osmo1)[0-9a-z]{38}$`
SIFCHAIN ADDR REGEXP `^(sif1)[0-9a-z]{38}$`

info by rango; also addr is null for gas tokens

**Review by mgmtea — APPROVED** [review]

File: src/utils/common.ts (hunk 1 above)
Comment (nrjshka) [reply src/utils/common.ts]: Fixed 🤝

**Review by insulineru — APPROVED** [review]
👍🏻

**Pull request merged**